packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -13,7 +13,7 @@ import {detectStorefrontPreview, getClientScript} from './utils'
 /**
  *
  * @param {boolean} enabled - flag to turn on/off Storefront Preview feature
- * @param {{function():string}} getToken - a STOREFRONT_PREVIEW customised function that fetches token of storefront
+ * @param {function(): string | Promise<string>} getToken - A method that returns the access token for the current user
```

Comment:
Let's match the changes to this jsdoc just like on the other PR #1440

Reply:
done.